libs/platform/src/lib/components/table/models/table-row.model.ts

```diff
@@ -21,6 +21,8 @@ export class TableRow<T = any> {
         // expanded/collapsed
         public expanded = true,
         // if item should be hidden. Used to skip rendering
-        public hidden = false
+        public hidden = false,
+        // if the row is navigatable
+        public navigatable = true
```

Comment:
Please follow JSDoc format for the comments so they are available for the IDE's

Reply:
I'll correct the other comments in this file along the way.